java/ql/lib/semmle/code/java/dataflow/ApiSinks.qll

```diff
@@ -0,0 +1,39 @@
+/** Provides classes representing various flow sinks for data flow / taint tracking. */
+
+private import semmle.code.java.dataflow.FlowSinks as FlowSinks
```

Comment:
See the next line:

```ql
final class SinkNode = FlowSinks::ApiSinkNode;
```

Without naming the import `as FlowSinks`, it would need to be written as:

```ql
final class SinkNode = ApiSinkNode;
```

Giving the imported module a name allows you to qualify anything it exports with that name. This is useful to avoid name clashes, for example.

Reply:
Ah, that makes sense. Thanks.